website/docs/docs/core/connect-data-platform/databricks-setup.md

```diff
@@ -48,17 +46,59 @@ pip is the easiest way to install the adapter:
 
 <p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
 
-`dbt-databricks` is the recommend adapter for Databricks
+`dbt-databricks` is the recommend adapter for Databricks.
 
 `dbt-databricks` includes features not available in `dbt-spark`:
 - Unity Catalog support
 - No need to install additional drivers or dependencies for use on the CLI
 - Use of Delta Lake for all models out of the box
 - SQL macros that are optimized to run with [Photon](https://docs.databricks.com/runtime/photon.html)
 
-### Set up a Databricks Target
+## Connecting to Databricks
+
+To connect to a data platform with dbt Core, create the appropriate _profile_ and _target_ YAML keys/values in the `profiles.yml` configuration file for your Databricks SQL Warehouse/cluster. This dbt YAML file lives in the  `.dbt/` directory of your user/home directory. For more information, refer to [Connection profiles](/docs/core/connect-data-platform/connection-profiles) and [profiles.yml](/docs/core/connect-data-platform/profiles.yml).
+
+`dbt-databricks` can connect to Databricks SQL Warehouses and all-purpose clusters. Databricks SQL Warehouses is the recommended way to get started with Databricks.
+
+
+Refer to the [Databricks docs](https://docs.databricks.com/dev-tools/dbt.html#) for more info on how
+to obtain the credentials for configuring your profile.
+
+## Host parameters
+
+The following profile fields are always required. 
+
+| Field     | Example | Description |
```

Comment:
@ericbaumann not sure if it's just me but i wonder if it would be easier to have the columns set as 'field', 'description', and 'example'. I'm finding myself scanning to the description first before example so i can understand the 'thing'. happy to change this up for you if you agree

Reply:
that's a great idea! Feel like that will decrease the cognitive load we put on the reader. Thank you @mirnawong1 !!